codegen/src/main/java/software/amazon/awssdk/codegen/poet/builder/BaseClientBuilderClass.java

```diff
@@ -721,6 +726,18 @@ private MethodSpec authSchemeProviderMethod() {
                          .build();
     }
 
+    private MethodSpec sigv4aRegionSetMethod() {
+        return MethodSpec.methodBuilder("sigv4aRegionSet")
```

Comment:
Should this just be `regionSet`? I guess we can finalize it in API surface area review

Reply:
Sure , I have this discussion during Surface API review